src/website/app/demos/Avatar/examples/avatar.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright 2017 CA
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* @flow */
+import Avatar from '../../../../../Avatar';
+import IconCloud from 'mineral-ui-icons/IconCloud';
+import DemoLayout from '../components/DemoLayout';
+
+export default {
+  id: 'basic',
+  title: 'Basic Usage',
+  description: `Avatar can display an image, a text character, or an
+[Icon](../icon). For accessibility, please provide an \`alt\`/\`title\`
```

Comment:
should alt/title be a required prop?

Reply:
The `alt`/`title` is on the `img`/Icon, so Avatar can't enforce that with the current API. `img` should always have an `alt`, even if simply empty, but Icon can have legitimate reason to not have a `title` (like when the title info would be redundant with the icon's text label).